pyproject.toml

```diff
@@ -43,6 +43,10 @@
 
 
 [tool.ruff.lint.isort]
-  combine-as-imports = true  # default is false
-
+  force-sort-within-sections = true
+  known-first-party = [
+      "homeassistant",
+  ]
```

Comment:
I'm not sure if this is a good idea or not. I can't find any generally accepted convention for custom components.